app/pipelines/pipelines/configuration.rb

```diff
@@ -44,7 +44,10 @@ def initialize(pipeline, children = {})
 
         children.each do |key, child|
           if child.instance_of?(ActiveSupport::HashWithIndifferentAccess)
-            create_instance_method(key) { Item.new(pipeline, child) }
+            unless respond_to?(key)
+              child_item = Item.new(pipeline, child)
+              create_instance_method(key) { child_item }
+            end
```

Comment:
Is this change still needed if we are using the `define_singleton_method`? If so, it would be worth bumping up the `respond_to?` to line 46 so we can remove the extra nested if.